### PR TITLE
Updates 20220517 (2nd pass)

### PR DESCRIPTION
**Pull request by willkg**
Dependabot created two more PRs and then I went through `requirements.in` and updated other things that were old:

Update dependencies. This covers:

* Reformat file
* black: 21.12b0 -> 22.3.0
* boto3: 1.20.26 -> 1.23.1
* botocore: 1.23.54 -> 1.26.1
* celery: 5.1.2 -> 5.2.6
* click: 7.1.2 -> 8.1.3
* datadog: 0.43.0 -> 0.44.0
* jsonschema: 4.4.0 -> 4.5.1
* markus: 4.0.0 -> 4.0.1
* symbolic: 8.7.0 -> 8.7.2
* urllib3: 1.26.8 -> 1.26.9
* Bump whitenoise from 6.0.0 to 6.1.0 (from PR #2553)
* Bump sentry-sdk from 1.5.10 to 1.5.12 (from PR #2554)

### CODE REVIEW

File: requirements.in [hunk 1]
```diff
@@ -1,12 +1,12 @@
 Sphinx==4.5.0
 backoff==2.0.1
-black==21.12b0
```
Comment (edugfilho) [inline requirements.in:3]: just curious: does b0 stand for something?
also because this is a major version update

**Review by edugfilho — APPROVED** [review]
looks good. Thanks!

File: requirements.in (hunk 1 above)
Comment (edugfilho) [reply requirements.in:3]: didn't look too much but couldn't find anything in their docs

**Comment by willkg** [comment]
Thank you!

File: requirements.in (hunk 1 above)
Comment (willkg) [reply requirements.in:3]: Until very recently, they were doing "pre-release" releases and only just switched to doing regular releases. Python version conventions is covered in pep-440 (https://peps.python.org/pep-0440/).

**Pull request merged**